server/src/test/java/org/spine3/server/entity/FieldMasksShould.java

```diff
@@ -0,0 +1,172 @@
+
+    @Test
+    public void have_private_constructor() {
+        assertTrue(hasPrivateUtilityConstructor(FieldMasks.class));
+    }
+
+    @Test
+    public void apply_mask_to_single_message() {
+        final FieldMask fieldMask = Given.fieldMask(Project.ID_FIELD_NUMBER, Project.NAME_FIELD_NUMBER);
+
+        final Project original = Given.newProject("some-string-id");
+
+        final Project masked = FieldMasks.applyMask(fieldMask, original, Given.TYPE);
+
+        assertNotEquals(original, masked);
+
+        assertMatchesMask(masked, fieldMask);
+    }
+
+    @SuppressWarnings("MethodWithMultipleLoops")
+    @Test
+    public void apply_mask_to_message_collections() {
+        final FieldMask fieldMask = Given.fieldMask(Project.STATUS_FIELD_NUMBER, Project.TASK_FIELD_NUMBER);
+        final int count = 5;
+
+        final Collection<Project> original = new LinkedList<>();
+
+        for (int i = 0; i < count; i++) {
+            final Project project = Given.newProject(String.format("project-%s", i));
+            original.add(project);
+        }
+
+        final Collection<Project> masked = FieldMasks.applyMask(fieldMask, original, Given.TYPE);
+
+        assertSize(original.size(), masked);
+
+        // Collection references are not the same
+        // noinspection ObjectEquality
+        assertFalse(original == masked);
+
+        for (Project project : masked) {
+            assertMatchesMask(project, fieldMask);
+
+            // Can't check repeated fields with assertMatchesMask
+            assertFalse(project.getTaskList()
+                               .isEmpty());
+        }
+    }
+
+    @Test
+    public void apply_only_non_empty_mask() {
+        // Empty mask
+        final FieldMask mask = Given.fieldMask();
+
+        final Project origin = Given.newProject("read_whole_message");
+        final Project clone = Project.newBuilder(origin)
+                                     .build();
+
+        final Project processed = FieldMasks.applyMask(mask, origin, Given.TYPE);
+
+        // Check object itself was returned
+        assertTrue(processed == origin);
+
+        // Check object was not changed
+        assertTrue(processed.equals(clone));
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void fail_to_mask_message_if_passed_type_dees_not_match() {
+        final FieldMask mask = Given.fieldMask(Project.ID_FIELD_NUMBER);
+
+        final Project origin = Given.newProject("some-string");
+
+        FieldMasks.applyMask(mask, origin, Given.OTHER_TYPE);
+    }
+
+    private static class Given {
+
+        private static final TypeUrl TYPE = TypeUrl.of(Project.class);
+        private static final TypeUrl OTHER_TYPE = TypeUrl.of(Customer.class);
+
+        private static Project newProject(String id) {
+            final ProjectId projectId = ProjectId.newBuilder()
+                                                 .setId(id)
+                                                 .build();
+
```

Comment:
Please avoid empty lines after the aligned chained calls. The code looks "spacy"-enough already.

To be applied here here and in all similar cases.